tst/EnergyPlus/unit/HVACUnitaryBypassVAV.unit.cc

```diff
@@ -204,12 +222,12 @@ TEST_F(EnergyPlusFixture, UnitaryBypassVAV_GetInputZoneEquipment)
         "AirLoopHVAC:UnitaryHeatCool:VAVChangeoverBypass,",
         "  Air Loop HVAC Unitary Heat Cool VAVChangeover Bypass 1, !- Name",
         "  ,                                       !- Availability Schedule Name",
-        "  0.02,                               !- Cooling Supply Air Flow Rate {m3/s}",
-        "  0.02,                               !- Heating Supply Air Flow Rate {m3/s}",
-        "  0.02,                               !- No Load Supply Air Flow Rate {m3/s}",
-        "  0.01,                               !- Cooling Outdoor Air Flow Rate {m3/s}",
-        "  0.01,                               !- Heating Outdoor Air Flow Rate {m3/s}",
-        "  0.01,                               !- No Load Outdoor Air Flow Rate {m3/s}",
+        "  0.021,                               !- Cooling Supply Air Flow Rate {m3/s}",
+        "  0.022,                               !- Heating Supply Air Flow Rate {m3/s}",
+        "  0.023,                               !- No Load Supply Air Flow Rate {m3/s}",
+        "  0.011,                               !- Cooling Outdoor Air Flow Rate {m3/s}",
+        "  0.012,                               !- Heating Outdoor Air Flow Rate {m3/s}",
+        "  0.013,                               !- No Load Outdoor Air Flow Rate {m3/s}",
```

Comment:
Just differentiating these so lines 445-450 can test that input was gotten correctly.